src/REGothEngine.hpp

```diff
@@ -42,27 +44,42 @@ namespace REGoth
 
       // Define engine options
       options.add_options()
-        ("a,game-assets", "Path to a Gothic or Gothic 2 installation", cxxopts::value<bs::Path>(assetsPath), "[PATH]")
+        ("a,game-assets", "Path to a Gothic or Gothic 2 installation", cxxopts::value<bs::Path>(originalAssetsPath), "[PATH]")
```

Comment:
Historically, the flag to specify the game root has always been `-g`. I think we should keep that!

Reply:
Great, didn't know that. Will do